dagger/bitswap/protobuf/payments.nim

```diff
@@ -0,0 +1,53 @@
+import pkg/protobuf_serialization
+import pkg/stew/byteutils
+import pkg/nitro
+import pkg/questionable
+import pkg/upraises
+import ./bitswap
+
+export PricingMessage
+export StateChannelUpdate
+
+export nitro
+
+push: {.upraises: [].}
```

Comment:
What's `upraises`?

Reply:
It's a little helper I wrote to deal with exception tracking in Nim 1.2.x. Full info here:  https://github.com/markspanbroek/upraises